setup.py

```diff
@@ -83,26 +83,15 @@
         ],
         'flask.commands': [
```

Comment:
Moderate: Since we are switching out of "app-context" and removing old commands. We can already move back to console scripts:

```
        'console_scripts': [
            'invenio-cli = invenio_cli.cli:cli',
        ],
```

Instead of `flask.commands`. Would close https://github.com/inveniosoftware/invenio-cli/issues/56.

Reply:
Thanks for that! Changed.